Core/Items/PoTGlobalItem.Data.cs

```diff
@@ -59,7 +60,12 @@ public override GlobalItem Clone(Item from, Item to)
 	/// <summary>
 	///		If the item is corrupt or not. Corrupted items cannot be modified with any currency shards.
 	/// </summary>
-	public bool Corrupted { get; set; } = false;
+	public bool Corrupted { get; set; }
+	
+	/// <summary>
+	///		If the item is corrupt or not. Corrupted items cannot be modified with any currency shards.
+	/// </summary>
+	public bool Cloned { get; set; } = false;
```

Comment:
This needs to be net synced in `PoTGlobalItem.IO.cs`. Just send the bool over like the `Corrupted` bool is, otherwise cloned items won't be synced properly (i.e. I clone an item, drop it to my friend, my friend will have an un-cloned item).

Reply:
If you want to be fancy, you could pack it into a byte too, if you want, but at 2 bools it's whatever.